draft-vvv-webtransport-http3.md

```diff
@@ -413,7 +413,7 @@ Setting Name:
 
 Value:
 
-: 0x???? (currently H2 draft uses 0xFB)
+: 0x63e941ac
```

Comment:
note that this is the 8-byte encoding not 4-byte :) Also please add the value to the [registry](https://github.com/quicwg/base-drafts/wiki/Temporary-IANA-Registry)